client/src/components/Card/CardResult.js

```diff
@@ -2,38 +2,55 @@ import React from "react";
 import PropTypes from "prop-types";
 import styled, { css } from "styled-components/macro";
 
+const barSize = {};
+
 const ResultCard = styled.div`
-  height: 100px;
-  width: 100px;
+  color: black;
+  margin-bottom: 2rem;
+
+  dd {
```

Comment:
what means dd?

Reply:
That's a html element especially used for charts - it has a default margin left and we thought it would perfectly fit in our use case. :)